src/components/AppProvider/utilities/withSticky/StickyManager.ts

```diff
@@ -1,6 +1,5 @@
-import {autobind} from '@shopify/javascript-utilities/decorators';
+import {autobind, debounce} from '@shopify/javascript-utilities/decorators';
```

Comment:
I'm curious why only this debounce was removed?

Reply:
`debounce` was added instead of `throttle`. The two functions are very similar and `throttle` can be achieved with `debounce` 😄